frontend/packages/dev-console/src/components/import/deploy-image/ResultsDisplay.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { Timestamp, units } from '@console/internal/components/utils';
+import { Alert } from '@patternfly/react-core';
+import { useFormikContext, FormikValues } from 'formik';
+
+const runsAsRoot = (image) => {
+  const user = _.get(image, 'dockerImageMetadata.Config.User');
+  return !user || user === '0' || user === 'root';
+};
+
+const ResultsDisplay: React.FC = () => {
+  const { values } = useFormikContext<FormikValues>();
+
+  const ImagePorts = ({ ports }) => (
+    <React.Fragment>
+      {_.size(ports) > 1 ? 'Ports ' : 'Port '}
+      {_.map(ports, (port) => `${port.containerPort}/${port.protocol.toUpperCase()}`).join(
+        ', ',
+      )}{' '}
+      will be load balanced by Service <strong>{values.name || '<name>'}</strong>.
+      <div>
+        Other containers can access this service through the hostname{' '}
+        <strong>{values.name || '<name>'}</strong>.
+      </div>
+    </React.Fragment>
+  );
+
+  return (
+    <div>
```

Comment:
Extra div seems unnecessary.

Reply:
I'm not sure why but when I remove it, I get an error. It doesn't like me to start with the next line.